File: modules/pose-camera/android/src/main/java/expo/modules/posecamera/PoseCameraModule.kt
package expo.modules.posecamera

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class PoseCameraModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("PoseCamera")

    Function("isAvailable") { true }

    View(PoseCameraView::class) {
      Events("onPose")
      Prop("facing") { view: PoseCameraView, facing: String -> view.facing = facing }
    }
  }
}

File: modules/pose-camera/ios/PoseCameraModule.swift
import ExpoModulesCore

public class PoseCameraModule: Module {
  public func definition() -> ModuleDefinition {
    Name("PoseCamera")

    Function("isAvailable") { true }

    View(PoseCameraView.self) {
      Events("onPose")
      Prop("facing") { (view: PoseCameraView, facing: String) in
        view.facing = facing
      }
    }
  }
}
